src/Tomlyn/Syntax/SyntaxNodeExtensions.cs

```diff
@@ -43,13 +43,19 @@ public static void Add(this SyntaxList<KeyValueSyntax> list, string name, int[]
             if (list == null) throw new ArgumentNullException(nameof(list));
             list.Add(new KeyValueSyntax(name, new ArraySyntax(values)));
         }
-
+        
         public static void Add(this SyntaxList<KeyValueSyntax> list, string name, string[] values)
         {
             if (list == null) throw new ArgumentNullException(nameof(list));
             list.Add(new KeyValueSyntax(name, new ArraySyntax(values)));
         }
 
+        public static void Add(this SyntaxList<KeyValueSyntax> list, string name, DateTimeValueSyntax value)
```

Comment:
Why not pass DateTime directly as the other methods?

Reply:
I initially did it like that but as TOML can output dates in 4 different formats (LocalDate, LocalDateTime, LocalTime, and OffsetDateTime) this leaves the choice still up to the user.
Should it also have one with DateTime that defaults to one specific format?